src/dpp/utility.cpp

```diff
@@ -550,6 +550,31 @@ namespace dpp {
 		    return "<@&" + std::to_string(id) + ">";
 		}
 
+		std::string message_url(const snowflake& guild_id, const snowflake& channel_id, const snowflake& message_id){
+			if (guild_id == snowflake(0) || channel_id == snowflake(0) || message_id == snowflake(0)) {
```

Comment:
you can just do !guild_id.empty() etc no need to compare to 0

Reply:
fixed

Reply:
still looks the same?